src/lib.rs

```diff
@@ -37,7 +37,10 @@ use zenoh::{
 const UATTRIBUTE_VERSION: u8 = 1;
 const THREAD_NUM: usize = 10;
 
-type CallbackChannelMessage = (Arc<dyn UListener>, Result<UMessage, UStatus>);
+struct CallbackChannelMessage {
```

Comment:
I like how the resulting code looks which interacts with `CallbackChannelMessage` now that it's a struct :+1: